templates/superset_python_path/superset_config.py.j2

```diff
@@ -13,8 +13,10 @@ from superset.stats_logger import DummyStatsLogger
 {% if superset_use_ketchup %}
 # patchup
 from superset_patchup.oauth import CustomSecurityManager
+from superset_patchup import patch_app
```

Comment:
With this hook in place I verified (manually) that the endpoints got installed on my test vmachine. 

As noted though, this will totally break if we don't push the superset-patchup stuff first.  Unclear what the approach is - the same breakage would occur for CustomSecurityManager, for example, if it wasn't already in.

Reply:
I think we [had added this](https://github.com/onaio/ansible-superset/blob/master/templates/superset_python_path/superset_config.py.j2#L141) in another PR.  Do you think that your change can go in that `{% if superset_flask_app_mutator %}` block?  One reason is that you may want to enable ketchup, but not necessarily the custom views.  Right?

Reply:
Hmm.. yeah, I guess in my mind this was organized a bit differently - you've got the choice "do I ketchup superset" - which is superset_use_ketchup - and then as a sub-choice you've potentially got "do I enable ketchup views".  In the patch you've shown, for example, superset_flask_app_mutator will enable ketchup views even if superset_use_ketchup is false.

I'd argue that the generic patch_app hook is more correct, and if we want to add additional parameters to that (like "patch_app(patch_views=True)") we can do that.  Basically my goal was to have as little control as possible over what got patched in this template file and the patching logic really lives in the patching code itself.  But I could totally be missing something, as this is my first PR :-).

> enable ketchup, but not necessarily the custom view
Yeah, thought about this, but it's kind of asymmetric in my head that enabling ketchup *always* enables the new security but then maybe wouldn't always enable new views?  Just thought a second pass to add flags for both might be cleaner, if required.

I actually think it'd be cool if we could push down the CustomSecurityManager registration too - so patch_app(patch_security=True, patch_views=True) - but figured that could be a second step.

Reply:
> Yeah, thought about this, but it's kind of asymmetric in my head that enabling ketchup always enables the new security but then maybe wouldn't always enable new views? Just thought a second pass to add flags for both might be cleaner, if required.

This is a good point.  We had it always enable the custom security because initially that was the only thing that Ketchup added.  But you are right, it need not be that way any more.

> I actually think it'd be cool if we could push down the CustomSecurityManager registration too - so patch_app(patch_security=True, patch_views=True) - but figured that could be a second step.

👍

> I'd argue that the generic patch_app hook is more correct, and if we want to add additional parameters to that (like "patch_app(patch_views=True)") we can do that. Basically my goal was to have as little control as possible over what got patched in this template file and the patching logic really lives in the patching code itself. But I could totally be missing something, as this is my first PR :-).

I generally agree with you, but it is not yet clear what the final config file will look like?  If we merge this as is, we'll have `FLASK_APP_MUTATOR` twice in the code, right?

Reply:
> but it is not yet clear what the final config file will look like?

Oh yeah, totally, I've got a logical merge conflict here (probably concurrent development? somehow I missed the views change) and need to resolve (probably also in superset-ketchup).  I'll work on that soon.